.github/workflows/docker-build.yaml

```diff
@@ -244,7 +314,8 @@ jobs:
                 github.event.pull_request.head.repo.full_name == 'shopsys/shopsys' ||
                 github.ref_protected == true
             ) &&
-            (needs.build-php-fpm-image.result == 'success' || needs.build-php-fpm-image.result == 'skipped')
+            (needs.build-php-fpm-image.result == 'success' || needs.build-php-fpm-image.result == 'skipped') &&
+            !contains(github.event.pull_request.labels.*.name, 'regenerate screenshots')
```

Comment:
this was not done deliberately for this scenario:
- you push changes
- screenshots fail
- you add a label
- you rerun the already created workflow

because the labels are evaluated at the time when the workflow is created, not when it is run, the screenshots will not be regenerated.